sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ReceiverOptions.java

```diff
@@ -27,10 +29,43 @@ ReceiveMode getReceiveMode() {
     }
 
     /**
-     * Gets the prefetch count of the receiver.
+     * Update enableSession flag.
```

Comment:
This doesn't need to be a mutable object. That's why the other fields are read-only.